### PR TITLE
[Workers for Platforms] Outbound Workers

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**
PCX-6886

### CODE REVIEW

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+# Outbound Workers
+
+Outbound Workers sit between your customer’s Workers and the public internet. They give you visibility into all outgoing `fetch()` requests from User Workers.
+
+![Outbound Workers diagram information](/images/cloudflare-for-platforms/outbound-worker-diagram.png)
+
+## General Use Cases
+
```
Comment (tanushree-sharma) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:13]: - Log all subrequests in order to identify malicious hosts or usage patterns
- Create allow or block lists for hostnames requested by user Workers
- Configure authentication to your APIs behind the scenes (without end developers needing to set credentials)

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+       // we tried to get a worker that doesn't exist in our dispatch namespace
+       return new Response('', { status: 404 });
+     }
+      return new Response(e.message, { status: 500 });
+   }
+ },
+};
+```
+
+4. The Outbound Worker will now be invoked on any `fetch()` request from User Workers.  This does not include `fetch()` through Service Bindings or mTLS certificate bindings.  The User Worker will trigger a [FetchEvent](/workers/runtime-apis/fetch-event/) on the Outbound Worker, with minimal performance overhead. The variables declared in the binding can be accessed in the Outbound Worker through `env.<VAR_NAME>`.
```
Comment (tanushree-sharma) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md]:
```suggestion
4. The Outbound Worker will now be invoked on any `fetch()` request from User Workers.  The User Worker will trigger a [FetchEvent](/workers/runtime-apis/fetch-event/) on the Outbound Worker, with minimal performance overhead. The variables declared in the binding can be accessed in the Outbound Worker through `env.<VAR_NAME>`. 
```
Can we add this line as a note at the bottom of the page - "This does not include `fetch()` through Service Bindings or mTLS certificate bindings. "

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Outbound Workers sit between your customer’s Workers and the public internet. They give you visibility into all outgoing `fetch()` requests from User Workers.
+
+![Outbound Workers diagram information](/images/cloudflare-for-platforms/outbound-worker-diagram.png)
+
+## General Use Cases
+
+### Use Outbound Workers
+
+To use Outbound Workers:
```
Comment (tanushree-sharma) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:22]: Can remove this line

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Create a Worker intended to serve as your Outbound Worker.
+2. Outbound Worker can be specified as an optional parameter in the [dispatch namespaces](/cloudflare-for-platforms/workers-for-platforms/get-started/configuration/#2-create-dispatch-namespace) binding in a projects [wrangler.toml](/workers/wrangler/configuration/). Optionally, to pass data from your Dispatch Worker to the Outbound Worker, the variable names must be specified under **parameters**.
```
Comment (tanushree-sharma) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:25]: @matthewdavidrodgers what wrangler version users need to update to in order to use the outbound?

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md (hunk 2 above)
Comment (Maddy-Cloudflare) [reply content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:62]: Does "this" refer to the request from User Workers?

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md (hunk 3 above)
Comment (Maddy-Cloudflare) [reply content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:22]: I would keep the subtitle. Here's why: 

1. The subtitle introduces instructions the user can access from the side menu (on the right).
2. The subsection after the subtitle gives instructions to the user on how to use Outbound Workers. 
3. It doesn't look like the subsection falls into any of the **General Use Cases**.

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md (hunk 4 above)
Comment (matthewdavidrodgers) [reply content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:25]: @tanushreesharma-cf can't answer that yet, as the the wrangler piece has not been merged (we were waiting for some pieces to fall into place from the backend). Will probably merged/release today

Comment (matthewdavidrodgers) [reply content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:25]: https://github.com/cloudflare/workers-sdk/pull/3200

Comment (matthewdavidrodgers) [reply content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md:25]: users need wrangler@3.0.0
https://github.com/cloudflare/workers-sdk/releases/tag/wrangler%403.0.0

File: content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+---
+pcx_content_type: concept
+title: Outbound Workers
+---
+
+# Outbound Workers
+
+Outbound Workers sit between your customer’s Workers and the public internet. They give you visibility into all outgoing `fetch()` requests from User Workers.
```
Comment (crwaters16) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/outbound-workers.md]:
```suggestion
Outbound Workers sit between your customer’s Workers and the public Internet. They give you visibility into all outgoing `fetch()` requests from User Workers.
```

**Review by crwaters16 — APPROVED** [review]

**Pull request merged**